test/testdata

```diff
@@ -231,6 +232,7 @@ SP        DALTON      GenericSPTest         basicDALTON-2013            dvb_sp_h
 SP        DALTON      GenericSPTest         basicDALTON-2013            dvb_sp_ks.out
 SP        DALTON      GenericSPTest         basicDALTON-2015            dvb_sp_hf.out
 SP        DALTON      GenericSPTest         basicDALTON-2015            dvb_sp_ks.out
+SP        Fchk        GenericSPTest         basicQChem5.2               dvb_sp_modified.FChk
```

Comment:
```suggestion
SP        FChk        GenericSPTest         basicQChem5.2               dvb_sp_modified.FChk
```
This is hiding a bunch of errors for me. I'm getting missing attribute errors for:
- atombasis
- atomcharges
- atommasses
- metadata["basis_set"]
- metadata["package_version"]
- moments
- mosyms
- scfenergies
- scftargets
- scfvalues

And some actual failures:
```
basicQChem5.2/dvb_sp_modified.FChk: Generic restricted single point unittest
testaooverlaps (test.data.testSP.GenericSPTest)Traceback (most recent call last):
  File "/home/eric/development/forks/cclib_pr_review/test/data/skip.py", line 18, in testwrapper
    testfunc(self, *args, **kwargs)
  File "/home/eric/development/forks/cclib_pr_review/test/data/skip.py", line 18, in testwrapper
    testfunc(self, *args, **kwargs)
  File "/home/eric/development/forks/cclib_pr_review/test/data/skip.py", line 18, in testwrapper
    testfunc(self, *args, **kwargs)
  [Previous line repeated 2 more times]
  File "/home/eric/development/forks/cclib_pr_review/test/data/testSP.py", line 220, in testaooverlaps
    self.assertEqual(self.data.aooverlaps[i,i], 1.0)
AssertionError: 9.35849267e-07 != 1.0
```
That has to be a parsing error, since I see lots of 1.0s. The error message is not great though, it would be better if that test extracted the entire diagonal from `aooverlaps`, since we can't tell with `i` it's failing on right now.
```
basicQChem5.2/dvb_sp_modified.FChk: Generic restricted single point unittest
testatomcoords_units (test.data.testSP.GenericSPTest)Traceback (most recent call last):
  File "/home/eric/development/forks/cclib_pr_review/test/data/testSP.py", line 79, in testatomcoords_units
    self.assertTrue(dev < 0.03, "Minimum carbon dist is %.2f (not 1.34)" % min_carbon_dist)
AssertionError: False is not true : Minimum carbon dist is 2.51 (not 1.34)
```
I think that one is just an angstrom/bohr problem.
```
basicQChem5.2/dvb_sp_modified.FChk: Generic restricted single point unittest
testhomos (test.data.testSP.GenericSPTest)Traceback (most recent call last):
  File "/home/eric/development/forks/cclib_pr_review/test/data/skip.py", line 18, in testwrapper
    testfunc(self, *args, **kwargs)
  File "/home/eric/development/forks/cclib_pr_review/test/data/testSP.py", line 141, in testhomos
    numpy.testing.assert_array_equal(self.data.homos, numpy.array([34],"i"), "%s != array([34],'i')" % numpy.array_repr(self.data.homos))
  File "/home/eric/.pyenv/versions/miniconda3-4.7.12/envs/cclib_37/lib/python3.7/site-packages/numpy/testing/_private/utils.py", line 931, in assert_array_equal
    verbose=verbose, header='Arrays are not equal')
  File "/home/eric/.pyenv/versions/miniconda3-4.7.12/envs/cclib_37/lib/python3.7/site-packages/numpy/testing/_private/utils.py", line 759, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not equal
array([34, 34], dtype=int32) != array([34],'i')
(shapes (2,), (1,) mismatch)
 x: array([34, 34], dtype=int32)
 y: array([34], dtype=int32)

basicQChem5.2/dvb_sp_modified.FChk: Generic restricted single point unittest
testmetadata_input_file (test.data.testSP.GenericSPTest)Traceback (most recent call last):
  File "/home/eric/development/forks/cclib_pr_review/test/data/skip.py", line 18, in testwrapper
    testfunc(self, *args, **kwargs)
  File "/home/eric/development/forks/cclib_pr_review/test/data/skip.py", line 18, in testwrapper
    testfunc(self, *args, **kwargs)
  File "/home/eric/development/forks/cclib_pr_review/test/data/skip.py", line 18, in testwrapper
    testfunc(self, *args, **kwargs)
  [Previous line repeated 9 more times]
  File "/home/eric/development/forks/cclib_pr_review/test/data/testSP.py", line 309, in testmetadata_input_file
    self.assertIn("input_file_contents", self.data.metadata)
AssertionError: 'input_file_contents' not found in {'package': 'FChk', 'methods': [], 'success': False}

basicQChem5.2/dvb_sp_modified.FChk: Generic restricted single point unittest
testmetadata_legacy_package_version (test.data.testSP.GenericSPTest)Traceback (most recent call last):
  File "/home/eric/development/forks/cclib_pr_review/test/data/testSP.py", line 330, in testmetadata_legacy_package_version
    self.assertIn("legacy_package_version", self.data.metadata)
AssertionError: 'legacy_package_version' not found in {'package': 'FChk', 'methods': [], 'success': False}
```